### PR TITLE
Update containers

### PR DESCRIPTION
**Pull request by yokaze**
Signed-off-by: Yusuke Suzuki <yusuke-suzuki@cybozu.co.jp>

### CODE REVIEW

File: calico/README.md [hunk 1]
```diff
@@ -15,11 +15,11 @@ Run the container
 ```console
 # Run as calico-node
 $ docker run -d --read-only --name=calico \
-    quay.io/cybozu/calico:3.14.1 start_runit
+    quay.io/cybozu/calico:3.16.1 start_runit
```
Comment (ymmt2005) [inline calico/README.md:18]: 3.16.1 does not exist.
```suggestion
    quay.io/cybozu/calico:3.16 start_runit
```

File: calico/README.md [hunk 2]
```diff
@@ -15,11 +15,11 @@ Run the container
...
 
 # Run as calico-typha
 $ docker run -d --read-only --name=calico --entrypoint="tini --"\
-    quay.io/cybozu/calico:3.14.1 calico-typha
+    quay.io/cybozu/calico:3.16.1 calico-typha
```
Comment (ymmt2005) [inline calico/README.md:22]: ditto

File: kubernetes/TAG [hunk 3]
```diff
@@ -1 +1 @@
-1.18.9.1
+1.18.9.2
```
Comment (ymmt2005) [inline kubernetes/TAG:1]: If you want to update this, update CKE too.
Personally, I don't think this needs to be updated now.

File: pause/TAG [hunk 4]
```diff
@@ -1 +1 @@
-3.2.0.1
+3.2.0.2
```
Comment (ymmt2005) [inline pause/TAG:1]: Don't update this for the same reason as kubernetes.

**Review by ymmt2005 — CHANGES_REQUESTED** [review]

File: kubernetes/TAG (hunk 3 above)
Comment (ysksuzuki) [reply kubernetes/TAG:1]: Just to be clear, this change is intended to update its ubuntu base image. Do we need to update it when we upgrade k8s to 1.19?

Comment (ymmt2005) [reply kubernetes/TAG:1]: I know.

> Do we need to update it when we upgrade k8s to 1.19?

I don't understand your question.  This *is* the Kubernetes.

File: calico/README.md (hunk 1 above)
Comment (ysksuzuki) [reply calico/README.md:18]: I think [calico v3.6.1](https://github.com/projectcalico/calico/releases/tag/v3.16.1) exits. Did I miss something?

Comment (ymmt2005) [reply calico/README.md:18]: @ysksuzuki It's the container image tag.

File: kubernetes/TAG (hunk 3 above)
Comment (ysksuzuki) [reply kubernetes/TAG:1]: Sorry, I don't understand your answer as well.

Comment (ymmt2005) [reply kubernetes/TAG:1]: To upgrade to Kubernetes 1.19, this _must_ be updated because this _is_ the Kubernetes.

File: pause/Dockerfile [hunk 5]
```diff
@@ -9,11 +9,11 @@ RUN mkdir /work
 
 WORKDIR /work
 
-RUN curl -sSLf -O https://github.com/kubernetes/kubernetes/raw/master/build/pause/pause.c \
+RUN curl -sSLf -O https://github.com/kubernetes/kubernetes/raw/719b3286635ad277865689a5a7cb5ce845a65daa/build/pause/pause.c \
     && gcc -Os -Wall -Werror -static -DVERSION=v${PAUSE_VERSION} -o pause pause.c \
     && strip pause
 
-RUN curl -sSLf -O https://github.com/kubernetes/kubernetes/raw/master/LICENSE
+RUN curl -sSLf -O https://github.com/kubernetes/kubernetes/raw/719b3286635ad277865689a5a7cb5ce845a65daa/LICENSE
```
Comment (ymmt2005) [inline pause/Dockerfile:16]: Please revert these changes.

**Review by ymmt2005 — CHANGES_REQUESTED** [review]

**Review by ymmt2005 — DISMISSED** [review]
LGTM

**Review by ysksuzuki — APPROVED** [review]
